src/microbe_stage/MicrobeAI.cs

```diff
@@ -114,6 +114,9 @@ public void Think(float delta, Random random, MicrobeAICommonData data)
     {
         _ = delta;
 
+        if (CheatMenu.Instance.NoAI)
```

Comment:
I think it would be much better to go to the top level code that runs the AI instead of doing stuff here, as this call is dispatched to separate threads so this wastes a ton of performance.